node/mo/pages/[...id].tsx

```diff
@@ -26,10 +58,14 @@ export async function getServerSideProps(context: any) {
   } else {
     // Search for regexs and replace :)
     let docs = await MoLink.aggregate([{ $match: { $expr: { $regexFind: { input: query, regex: "$alias" } } } }]).exec()
-    if (docs.length > 0) {
+    if (docs.length == 1) {
       let link = docs[0]
       await MoLink.updateOne({ _id: link._id }, { $inc: { n: 1 } }).exec()
       url = query.replace(new RegExp(link.alias), link.link)
+    } else if (docs.length > 1) {
+      return {
+        props: {url: null , links:docs.map(link =>JSON.parse(JSON.stringify({_id:link._id.toString(), alias:link.alias, n:link.n, link:link.link})))},
```

Comment:
`JSON.parse(JSON.stringify(...))` seems odd here?

Reply:
Can this just be: `props: {url: null , links: docs}`?

Reply:
You are right. I could find a proper JS way to convert it to an object. `{url: null , links: docs}` does not work for me. I'm still training my javascript-fu.

Reply:
Just using: `{_id:link._id.toString(), alias:link.alias, n:link.n, link:link.link}` should work here.